src/model.rs

```diff
@@ -700,34 +728,38 @@ impl Component for Model {
             }
 
             Action::ReadSession(file) => {
-                let callback = self.link.callback(Action::LoadSession);
-                let task = self.reader.read_file(file, callback);
-                self.tasks.push(task);
+                info!("in here");
```

Comment:
remember to take out these log statements before merging!!

Reply:
I worked on my save-loading branch that's why there is this comment still, I ll merge this before that one so it should be fine